.github/workflows/zetachain-cctx.yml

```
@@ -0,0 +1,93 @@
+      working-directory: zetachain-cctx
+
+  docker:
+    name: Docker build and docker push
+    needs:
+      - test
+      - lint
+    if: |
+      always() &&
+      (needs.test.result == 'success' || needs.test.result == 'cancelled') &&
+      (needs.lint.result == 'success' || needs.lint.result == 'cancelled')
+    uses: ./.github/workflows/_docker-build-push.yml
```

actionlint (1.7.7)

secret "ARM_RUNNER_HOSTNAME" is required by "./.github/workflows/_docker-build-push.yml" reusable workflow

(workflow-call)